cmd/cmd_test.go

```diff
@@ -441,21 +441,39 @@ func TestDiffRevisions(t *testing.T) {
 	run.MustExec(t, "qri save --body=testdata/movies/body_thirty.csv me/test_movies")
 	output := run.MustExec(t, "qri diff body me/test_movies")
 
-	expect := `+30 elements. 30 inserts. 0 deletes. 0 updates.
-
-+ 18: ["Dragonfly ",104]
-+ 19: ["The Black Dahlia ",121]
-+ 20: ["Flyboys ",140]
-+ 21: ["The Last Castle ",131]
-+ 22: ["Supernova ",91]
-+ 23: ["Winter's Tale ",118]
-+ 24: ["The Mortal Instruments: City of Bones ",130]
-+ 25: ["Meet Dave ",90]
-+ 26: ["Dark Water ",103]
-+ 27: ["Edtv ",122]
+	expect := `+30 elements. 10 inserts. 0 deletes.
+
+ 0: ["Avatar ",178]
+ 1: ["Pirates of the Caribbean: At World's End ",169]
+ 2: ["Spectre ",148]
+ 3: ["The Dark Knight Rises ",164]
+ 4: ["Star Wars: Episode VII - The Force Awakens             ",""]
+ 5: ["John Carter ",132]
+ 6: ["Spider-Man 3 ",156]
+ 7: ["Tangled ",100]
+ 8: ["Avengers: Age of Ultron ",141]
+ 9: ["Harry Potter and the Half-Blood Prince ",153]
+ 10: ["Batman v Superman: Dawn of Justice ",183]
+ 11: ["Superman Returns ",169]
+ 12: ["Quantum of Solace ",106]
+ 13: ["Pirates of the Caribbean: Dead Man's Chest ",151]
+ 14: ["The Lone Ranger ",150]
+ 15: ["Man of Steel ",143]
+ 16: ["The Chronicles of Narnia: Prince Caspian ",150]
+ 17: ["The Avengers ",173]
++18: ["Dragonfly ",104]
```

Comment:
What do you think about adding a TODO to limit the amount of context shown? I feel 4 or 5 lines is plenty to understand the position where the file is changing, which is similar to what Unix Diff does. Any more than that and it gets hard to read.

Reply:
Agreed. I think this would be best filed as a feature request issue on deepdiff, as that's where it needs to originate from. I'd like to be able to arbitrarily set the amount of context, but for that to work we need to flesh out the spec for "jump points" that skip rows in the patch.